components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApisApiServiceImpl.java

```diff
@@ -1009,6 +1015,37 @@ public Response apisApiIdSubscriptionPoliciesGet(String apiId, String organizati
         return null;
     }
 
+    @Override
+    public Response getIdpsOfAPI(String apiId, MessageContext messageContext) {
+        try {
+            APIConsumer apiConsumer = RestApiCommonUtil.getLoggedInUserConsumer();
+            String organization = APIUtils.getOrganization(messageContext);
```

Comment:
there is one method in RestApiAdminUtils to get this, shall we unify  this

Reply:
Fixes in 32aae8fa15ab3a07e2e6896a1f702cfd91df1f05